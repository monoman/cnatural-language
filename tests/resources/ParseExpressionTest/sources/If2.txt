() => { if (true) return false; return true; }