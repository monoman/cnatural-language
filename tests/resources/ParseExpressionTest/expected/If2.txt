Lambda( => {
if (Literal(True, [true]))
return(Literal(False, [false]));
return(Literal(True, [true]));
}
)